docs/changelog/overview.mdx

```diff
@@ -4,6 +4,17 @@ title: "Changelog"
 
 The changelog below reflects new product developments and updates on a monthly basis.
 
+## January 2025
+- Added [Secret Syncs](https://infisical.com/docs/integrations/secret-syncs/overview) for AWS Parameter Store, GitHub, and GCP Secret Manager
```

Comment:
some of these have dots at the end, while the others don't. I think we should have dots everywhere to stay consistent.